### ISSUE TITLE
Accessibility options 

### ISSUE THREAD

**Issue by Katfox2004** [ControlNet/wt-data-project.web#19]
For someone like me who is colour blind and has a hard time telling dark colours apart, it would be nice to be able to have an option to change the colours of the scale. Like navy blue to orange or smth like that.

**Comment by ControlNet** [comment]
Hi, I think it's a really good suggestions. However, I have no idea about the colors used for color blind mode. As you suggested, I may start from navy blue to orange or smth.

I will update some preview when it's ready.

**Comment by ControlNet** [comment]
The webpage is updated with a shiny new checkbox. You can switch the mode by checking it. 

How's it looks like now?
![image](https://github.com/ControlNet/wt-data-project.web/assets/12800094/069d720c-2b57-4784-beab-d9ffd727bc39)

**Comment by RealmX1** [comment]
Not sure how each individual colorblindness work exactly, but Greyscale will be an easy adaptation for multiple types of color blindness.

**Comment by Corn-on-the-Job** [comment]
Yeah I second the greyscale idea, simplifies correcting for the different types of colorblindness. I'm not certain but it looks like the current implementation is probably correcting for tritanopia, which is the least common of the standard trichromatic color defficiencies. I'm moderately protanopic and the all purples is harder to parse than the default one.

**Comment by MeanGreeny** [comment]
**Hi,

Adding to the greyscale suggestion, the various types of colour blindness really make a change in grey tone the best option to cater for all the various conditions. White for 0-5 and black for 95-100 allows for easy interpretation of a heatmap by all of the various forms of sufferers [about 15% of all males].

The more tones the better - as a minimum, 5 percent increments [20 tonal values] if you can manage it.

Fingers crossed that you can find the time**

**Comment by webbrain-one** [comment]
Thanks for highlighting this! I’ve opened [PR #32](https://github.com/ControlNet/wt-data-project.web/pull/32), which adds a colour option to make the scale easier to distinguish for colour-blind users, e.g. switching darker tones to a high-contrast orange-based palette.
